src/app/static/question-filter/question-filter.component.ts

```diff
@@ -129,18 +129,21 @@ export class QuestionFilterComponent implements OnInit {
     }
   }
 
-  selected(event: MatAutocompleteSelectedEvent): void {
-    this.tags.push(event.option.viewValue);
-    this.tagInput.nativeElement.value = '';
-    this.filterInputTags();
-    this.tagCtrl.setValue(null);
-  }
+
+selected(event: MatAutocompleteSelectedEvent): void {
+  this.tags.push(event.option.viewValue);  
+  this.tagInput.nativeElement.value = '';
+  this.filterInputTags();
+  this.tagCtrl.setValue(null);
+}
+
```

Comment:
Please indent things inside a class, and match existing style rather than change it (unless the change is a clear improvement)

Reply:
Fixed